sdks/go/pkg/beam/io/healthcare/fhirio/fakes.go

```diff
@@ -0,0 +1,40 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fhirio
+
+import (
+	"net/http"
+)
+
+type fakeFhirStoreClient struct {
+	fakeReadResources func(string) (*http.Response, error)
+}
+
+func (c *fakeFhirStoreClient) readResource(resourceName string) (*http.Response, error) {
+	return c.fakeReadResources(resourceName)
+}
+
+type fakeReaderCloser struct {
```

Comment:
Confused by the naming/purpose for this struct, can you rename or add a comment.

Reply:
Added a comment. It is being used [here](https://github.com/lnogueir/beam/blob/98b928607168dfe265d76ac8655317f070e8400c/sdks/go/pkg/beam/io/fhirio/read_test.go#L56).
Hopefully it makes sense now.

Reply:
Looks good.